exporting/prometheus/multi_metadata.yaml

```diff
@@ -0,0 +1,436 @@
+    exporter_limitations: 'The remote write exporting connector does not support buffer on failures.'
+  setup:
+    prerequisites:
+      list:
+        - title: ''
+          description: |
+            - Netdata and the external storage provider of your choice, installed, configured and operational.
+            - `protobuf` and `snappy` libraries installed.
+            - Netdata reinstalled after the libraries.
+    configuration:
+      file:
+        name: 'exporting.conf'
+      options:
+        description: 'The following options can be defined for this exporter.'
+        folding:
+          title: 'Config options'
+          enabled: true
+        list:
+          - name: 'enabled'
+            default_value: 'no'
+            description: 'Enables or disables an exporting connector instance (yes|no).'
+            required: true
+          - name: 'destination'
+            default_value: 'no'
+            description: 'Accepts a space separated list of hostnames, IPs (IPv4 and IPv6) and ports to connect to. Netdata will use the first available to send the metrics.'
+            required: true
+            detailed_description: |
+              The format of each item in this list, is: [PROTOCOL:]IP[:PORT].
+              - PROTOCOL can be udp or tcp. tcp is the default and only supported by the current exporting engine.
+              - IP can be XX.XX.XX.XX (IPv4), or [XX:XX...XX:XX] (IPv6). For IPv6 you can to enclose the IP in [] to separate it from the port.
+              - PORT can be a number of a service name. If omitted, the default port for the exporting connector will be used.
+  
+              Example IPv4:
+                ```yaml
+                destination = 10.11.14.2:4242 10.11.14.3:4242 10.11.14.4:4242
+                ```
+              Example IPv6 and IPv4 together:
+              ```yaml
+              destination = [ffff:...:0001]:2003 10.11.12.1:2003
+              ```
+              When multiple servers are defined, Netdata will try the next one when the previous one fails.
+          - name: 'username'
+            default_value: 'my_username'
+            description: 'Username for HTTP authentication'
+            required: false
+          - name: 'password'
+            default_value: 'my_password'
+            description: 'Password for HTTP authentication'
+            required: false
+          - name: 'data source'
+            default_value: ''
+            description: 'Selects the kind of data that will be sent to the external database. (as collected|average|sum)'
+            required: false
+          - name: 'hostname'
+            default_value: '[global].hostname'
+            description: 'The hostname to be used for sending data to the external database server.'
+            required: false
+          - name: 'prefix'
+            default_value: 'Netdata'
```

Comment:
The default value is in lowercase.

Reply:
was also finding it strange this being capitalized but when to check [our docs](https://learn.netdata.cloud/docs/exporting/exporting-reference#options) and is there like this
![image](https://github.com/netdata/netdata/assets/82235632/8ce04bff-421c-4d3c-b1ed-be492ee60178)

our docs should be driven by the README on the repo so the mistake could be there